.reek

```diff
@@ -28,6 +28,7 @@ UnusedPrivateMethod:
     - ActiveJob::Logging::LogSubscriber
     - Users::PhoneConfirmationController
 UtilityFunction:
+  public_methods_only: true
```

Comment:
Maybe just exclude that classes in question?  I'm not sure the smell is any different for private vs public methods is it?